### ISSUE TITLE
importing not included concepts

### ISSUE THREAD

**Issue by luisenriqueramos1977** [ontodev/robot#1062]
Dear All, 

I am extracting a module from chebi ontology, using the following command:
robot extract --method STAR --input source_ontos/chebi.owl --term-file onto_terms/chebi_terms.txt --output extracted_modules/start_method/chebi_start.owl
and the following terms:

http://purl.obolibrary.org/obo/CHEBI_24431
http://purl.obolibrary.org/obo/CHEBI_24433
http://purl.obolibrary.org/obo/CHEBI_33250

However the terms **atomic nucleus, electron, and nucleon** are being imported, which are not subclasses of the concepts listed above.
In other words, I am successfully importing the terms chemical entity, atom, and group, but additional terms, which are not a subclass of chemical entity are being imported.

Thus, I wonder where should be the error from my side?

Best regards

Luis Ramos

**Comment by matentzn** [comment]
@luisenriqueramos1977 

Any [SLME module](https://robot.obolibrary.org/extract#syntactic-locality-module-extractor-slme) will extract all terms logically related to  the input signature. STAR is the smallest, but it will still import a lot of terms. If you want to be 100% minimal use ROBOT filter.

**Comment by jamesaoverton** [comment]
I would suggest trying the [MIREOT](http://robot.obolibrary.org/extract#mireot) method of `robot extract` first. `robot filter` is also an option, but is more difficult to configure. I believe that @matentzn is thinking of the [ODK](https://github.com/INCATools/ontology-development-kit) has a `filter` module type.
